keps/prod-readiness/sig-network/2079.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@wojtek-t"
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@johnbelamaric"
```

Comment:
Please reassign back to me - we're generally trying not to change PRR approver in the middle due to having a context.

And I'm happy to take it.

Reply:
@wojtek-t sure will do! I changed to John per the prr-reviewers spreadsheet, but will rollback today

Reply:
thanks